ui/job-view/PushActionMenu.jsx

```diff
@@ -136,7 +136,7 @@ export default class PushActionMenu extends React.PureComponent {
             target="_blank"
             rel="noopener noreferrer"
             className="dropdown-item"
-            href={`https://bugherder.mozilla.org/?cset=${revision}&amp;tree=${repoName}`}
+            href={`https://bugherder.mozilla.org/?cset=${revision}&tree=${repoName}`}
```

Comment:
Yeah looks like only direct string literals are unescaped:
https://reactjs.org/docs/jsx-in-depth.html#string-literals